btx/diagnostics/run.py

```diff
@@ -137,12 +162,24 @@ def compute_stats(self, evt, img):
                       "Speak with a beamline scientist to find the proper PV")
             self.stats['photon_energy_eV'][self.n_proc] = photon_energy_eV
 
-            self.stats['mean'][self.n_proc] = np.mean(img)
-            self.stats['std'][self.n_proc] = np.std(img)
-            self.stats['max'][self.n_proc] = np.max(img)
-            self.stats['min'][self.n_proc] = np.min(img)
+            if img is None:
+                self.stats['mean'][self.n_proc] = np.nan
```

Comment:
Is `nan` or `None` most appropriate here? I think there can be some subtle differences between the two which may cause issues down the line. The two that come to mind are `np.nan != np.nan` and that `np.nan` evaluates as Truthy in conditionals.

Reply:
Hmmm I didn't consider this. Would plotting of `None` still work?

Reply:
We checked and `None` wouldn't work with the `color_by` interactive display option.